content/embeds/cluster-setup.md

```diff
@@ -0,0 +1,26 @@
+1. In the web browser on the host machine, go to https://localhost:8443 to see
+the Redis Enterprise Software web console.
+
+    Note:
+
+    - Depending on your browser, you may see a certificate error. You can safely
+    continue to the web console.
+    - If you see an error from nginx, try again after a few minutes.
+
+1. Click **Setup** to start the node configuration steps.
+
+    ![Redis Enterprise Software Setup](/images/rs/getstarted-setup.png?width=600)
+
+1. In the **Node Configuration** settings, enter a cluster FQDN such as `cluster.local`.
+Then click **Next** button.
+
+    ![Redis Enterprise Software node configuration](/images/rs/getstarted-nodeconfig.png?width=600)
+
+1. Enter your license key, if you have one. If not, click the **Next** button to use the trial version.
+
+1. Enter an email and password for the admin account for the web console.
```

Comment:
Actually it's an Admin for the UI as well as for the cluster.
Meaning it's also the Admin user to send API command with its credentials

Reply:
Added "These credentials are also for the connections to the REST API." OK?

Reply:
how about "These credentials are also to use the Redis REST API."?